variables.tf

```
@@ -361,6 +361,12 @@ variable "runners_output_limit" {
   default     = 4096
 }
 
+variable "runners_wait_for_services_timeout" {
+  description = "How long to wait for Docker services. Set to -1 to disable. Default is 30."
+  type        = string
```

question (blocking): Any reason to choose a `string`? Looks like a number.
```suggestion
  type        = number
```

---

I wasn't 100% sure how Terraform would handle `-1` since this is a valid value for the runner config to disable the feature. But since I was curious, I've tested it myself and indeed number is the correct type here. I will change it ASAP.